wiki/Tournaments/OTC/3/en.md

```diff
@@ -0,0 +1,307 @@
+---
+tags:
+- 'OTC #3'
+- OTC 3
+- OTC#3
+- OTC3
+---
+# osu! Technical Championship #3
+
+![OTC #3 Logo](img/logo.png)
+
+The **osu! Technical Championship #3** (***OTC #3***) is a double elimination one-on-one osu! tournament hosted by ![][flag_US] [IsoSubjectPi](/users/3989669). osu! Technical Championship features complex rhythms and sliders, unorthodox patterns, and a wide variety of technical mapping in its mappools. It is the 3rd installment of the osu! Technical Championship.
```

Comment:
```suggestion
The **osu! Technical Championship #3** (***OTC #3***) is a double-elimination one-on-one osu! tournament hosted by ![][flag_US] [IsoSubjectPi](/users/3989669). osu! Technical Championship features complex rhythms and sliders, unorthodox patterns, and a wide variety of technical mapping in its mappools. It is the third installment of the osu! Technical Championship.
```
same as last time :wink: i should probably change the other tournament articles sometime...